src/nvim/quickfix.c

```diff
@@ -3515,22 +3519,13 @@ int set_errorlist(win_T *wp, list_T *list, int action, char_u *title)
         did_bufnr_emsg = TRUE;
         EMSGN(_("E92: Buffer %" PRId64 " not found"), bufnum);
       }
-      valid = FALSE;
+      valid = false;
       bufnum = 0;
     }
 
-    status =  qf_add_entry(qi, &prevp,
-        NULL,                               /* dir */
-        filename,
-        bufnum,
-        text,
-        lnum,
-        col,
-        vcol,                               /* vis_col */
-        pattern,                            /* search pattern */
-        nr,
-        type == NULL ? NUL : *type,
-        valid);
+    int status = qf_add_entry(qi, &prevp, NULL, filename, bufnum,
+                              text, lnum, col, vcol, pattern, nr,
+                              (char_u)(type == NULL ? NUL : *type), valid);
```

Comment:
I prefer the old version that includes the comments and our style guide recommends that as well.

> On one line if it fits; otherwise, wrap arguments at the parenthesis. 
https://neovim.io/develop/style-guide.xml?showone=Function_Calls#Function_Calls

Please restore the comments and I will merge this. :)